### ISSUE TITLE
Nothing happens after loading weights for yolov4.cfg

### ISSUE THREAD

**Issue by alemyewno** [AlexeyAB/darknet#8167]
I trained yolov4-tiny.cfg on one class and that worked. But, when I wanted to train yolov4.cfg, after loading weights it just put it on hold. I suppose it is not because of cuda because it works for the tiny version. I suppose the problem is because of something in yolov4.cfg. This is what I get:

`[yolo] params: iou loss: ciou (4), iou_norm: 0.07, obj_norm: 1.00, cls_norm: 1.00, delta_norm: 1.00, scale_x_y: 1.05
Total BFLOPS 59.563 
avg_outputs = 489778 
 Allocate additional workspace_size = 52.43 MB 
Loading weights from yolov4.weights...Done! Loaded 162 layers from weights-file 
`
Here it just holds. So, the script is not finishing the process. It is just waiting something. This is the beginning of my yolov4.cfg file: 
`
[net]
batch=1
subdivisions=1
# Training
#width=512
#height=512
width=416
height=416
channels=3
momentum=0.949
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.0013
burn_in=1000
max_batches = 20000
policy=steps
steps=16000,18000
scales=.1,.1

#cutmix=1
mosaic=1

#:104x104 54:52x52 85:26x26 104:13x13 for 416
`
I changed the "filters" part to be 18 because there is only one class present here. So, everything loaded, but it just waits something. yolov4-tiny.cfg works fine, but this one doesn't. What could be the cause of this? My command is this 

`./darknet detector train data/obj.data cfg/yolov4.cfg  yolov4.weights -dont_show -mjpeg_port 8090 > log.txt -map
`

**Comment by alemyewno** [comment]
@bulatnv Well, the same command with tiny version works. I just changed the model and everything is the same. Paths are ok because I used proper commands to fill train.txt and val.txt. For example `find `pwd`/train -name \*.jpg > train.txt` I did it once and tiny version worked perfectly, so I didn't change it. I just change the model.

**Comment by alemyewno** [comment]
@bulatnv I don't have problems with that command. Well, I don't know much about subdivisions. I put 1 but I don't know what is good for that.
![Screenshot from 2021-10-20 14-17-13](https://user-images.githubusercontent.com/90051878/138091114-98568bf6-2eb0-431a-8359-4614c256ef0b.png)

**Comment by bulatnv** [comment]
@alemyewno 
It is realy hard to understand what is going wrong. Can you provide more information:
1) System (Hardware, Software)
2) Run command
3) Attach cfg as file.

**Comment by alemyewno** [comment]
@bulatnv I changed width and height 416 to 320 and now it works. It was probably a memory issue.
